Module-generation/scrape.py

```diff
@@ -0,0 +1,120 @@
+import os
+import json
+import requests
+from github import Github
+
+IndexDir="./meta-data/"
+os.mkdir(IndexDir)
+
+accessToken=os.environ.get('GIT_CREDS')
+g = Github(accessToken)
+
+
+user= g.get_user('Terasology')
+repo = user.get_repo("Index")
+
+contents = repo.get_contents("/distros/omega/gradle.properties")
+x=contents.decoded_content.decode()
+
+# Filter Modules and Convert it to array
+f = open('Modules.txt', 'w')
+f.write(x)
+f.close()
+f = open("Modules.txt", "r")
+lines=f.readlines()
+line=lines[1]
+word1='extraModules='
+word2='\n'
+wordreplace=line.replace(word1, "")
+wordreplace=wordreplace.replace(word2,"")
+Modules=wordreplace.split(",")
```

Comment:
Why do we need to write this to a file and then read and process it from there.
I don't think this is necessary and you can parse the response directly.

Reply:
Done